run_terrier_comparison.py の必須引数不足エラー

GitHub Actions の `manual-workflows.yml` ワークフローで `scripts/run_terrier_comparison.py` を呼び出す際に、必須の位置引数である `image1_path` と `image2_path` が指定されていません。

**エラーメッセージ:**
```
usage: run_terrier_comparison.py [-h] image1_path image2_path
run_terrier_comparison.py: error: the following arguments are required: image1_path, image2_path
Error: Process completed with exit code 2.
```

**発生箇所:**
`.github/workflows/manual-workflows.yml` の81行目

**解決策の提案:**
`manual-workflows.yml` 内の `run_terrier_comparison.py` の呼び出しを修正し、2つの有効な画像パスを位置引数として指定してください。

例:
`python scripts/run_terrier_comparison.py sigma_images/circle_center.jpg sigma_images/square_left.jpg`

--- a/.github/workflows/manual-workflows.yml
+++ b/.github/workflows/manual-workflows.yml
@@ -78,6 +78,6 @@ jobs:
     - name: Generate AI dimensions
       run: python src/generate_ai_dimensions.py selia
     - name: Run terrier comparison script
-      run: python scripts/run_terrier_comparison.py
+      run: python scripts/run_terrier_comparison.py sigma_images/circle_center.jpg sigma_images/square_left.jpg
     - name: Run psyche simulation script
       run: python scripts/run_psyche_simulation.py

